packages/nx-sonarqube/package.json

```diff
@@ -11,10 +11,10 @@
   "generators": "./generators.json",
   "executors": "./executors.json",
   "peerDependencies": {
-    "@nx/devkit": ">= 16 < 18",
+    "@nx/devkit": ">= 17 < 19",
```

Comment:
I don't believe there any breaking changes between 16 & 18, but let me know if I'm incorrect. Otherwise, change both `@nx/devkit` and `nx` deps to have a range of `>= 16 < 19`